app/controllers/feedback_controller.rb

```diff
@@ -9,6 +9,7 @@ def admin
 
   def new
     @feedback = Feedback.new
+    session[:subject] = params[:subject] || "Dispatch"
```

Comment:
We should default to just "Caseflow", I think— why did we change this to "Dispatch"?